src/bin/inx-chronicle/launcher.rs

```diff
@@ -77,18 +77,38 @@ impl Actor for Launcher {
 
         #[cfg(feature = "stardust")]
         {
-            db.collection::<chronicle::db::collections::OutputCollection>()
+            use chronicle::db::collections;
+            let start_indexes = db.get_index_names().await?;
```

Comment:
One additional question: is it necessary that `get_index_names()` comes before the `create_collection` calls? Otherwise it would probably make sense to move it right before the index creation logic.

Reply:
I believe that it is necessary, because creating the collections creates `_id` indexes.